<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EAbonnement</class>
 <widget class="QWidget" name="EAbonnement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/Etudiant/Etudiant Abonnement.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Classe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Notification">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>910</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="CoursTab">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>1260</width>
     <height>630</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Afficher Abonnement</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherA">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>165</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID Abonnement</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prix</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Debut</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Fin</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duree</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Personne</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="Afficher">
    <attribute name="title">
     <string>Afficher Cours</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherCo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>921</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>181</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>28</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Horaire</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NBR Reservation</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="NBRR_3">
     <property name="geometry">
      <rect>
       <x>1320</x>
       <y>50</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>210</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/Reserver.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Reserver">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>200</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Commander Produit</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherST">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>891</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>220</number>
     </attribute>
     <column>
      <property name="text">
       <string>Reference</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prix_Unitaire</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Produit</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantite</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>200</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/commander.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Commander">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>190</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
